src/Bridge/Doctrine/Orm/SubresourceDataProvider.php

```diff
@@ -63,6 +64,7 @@ public function __construct(ManagerRegistry $managerRegistry, PropertyNameCollec
      * {@inheritdoc}
      *
      * @throws RuntimeException
+     * @throws ItemNotFoundException
      */
     public function getSubresource(string $resourceClass, array $identifiers, array $context, string $operationName = null)
```

Comment:
It's not the dataprovider's responsability to throw an `ItemNotFoundException`. To stay constistent with `ItemDataProvider`, please patch this inside the [`ReadListener`](https://github.com/api-platform/core/blob/master/src/EventListener/ReadListener.php#L120) instead.

Reply:
done